app/views/admin/sessions/new.html.haml

```diff
@@ -5,7 +5,7 @@
 
 .govuk-grid-row
   .govuk-grid-column-two-thirds
-    = link_to 'Start now', auth_provider_path, class: 'govuk-button govuk-button--start', id: 'start-now'
+    = button_to 'Start now', auth_provider_path, class: 'govuk-button govuk-button--start', id: 'start-now'
```

Comment:
What's with the `button_to` change?

Reply:
It's because the vulnerability is based on the fact that omniauth allows the auth to be initiated by a GET request, which `omniauth-rails_csrf_protection` stops, so we need to make it a POST

Reply:
Ah, sorry, yes, I found this while reading the `omniauth-rails_csrf_protection` README :joy: